features/support/initial_data.rb

```diff
@@ -13,13 +13,11 @@
   ThreeScale.config.stubs(superdomain: '3scale.localhost')
 
   SphinxIndexationWorker.stubs(:perform_later)
-  IndexProxyRuleWorker.stubs(:perform_later)
 
   FieldsDefinition.create_defaults! master_account
 
   if scenario.source_tag_names.include?('@search')
     SphinxIndexationWorker.unstub(:perform_later)
```

Comment:
```suggestion
  SphinxIndexationWorker.unstub(:perform_later) if scenario.source_tag_names.include?('@search')
```

as CodeClimate suggests

NOTE that the suggestion can't be applied as it is, because it would leave the orphaned `end`